src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -303,8 +303,11 @@ public with sharing class CRLP_RollupUI_SVC {
         Set<String> excludedTypes = new Set<String>{'TIME', 'TEXTAREA'};
 
         for(String obj : objectNames){
-            if (obj == 'Partial_Soft_Credit__c') {
-                List<String> fieldsToReturn = new List<String>{'Role_Name__c','Amount__c'};
+            if (obj == UTIL_Namespace.StrAllNSPrefix('Partial_Soft_Credit__c')) {
```

Comment:
@bethbrains We don't need to fix this here given the limited usage, but in general it would be better to assign UTIL_Namespace.StrAllNSPrefix('Partial_Soft_Credit__c') to a var outside the loop rather than call the utility method for each loop iteration.